packages/@aws-cdk/aws-rds/lib/private/instance-deletion-policy.ts

```diff
@@ -0,0 +1,13 @@
+import { CfnDeletionPolicy, RemovalPolicy } from '@aws-cdk/core';
+import { CfnDBInstance } from '../rds.generated';
+
+export function applyInstanceDeletionPolicy(cfnDbInstance: CfnDBInstance, removalPolicy: RemovalPolicy | undefined): void {
```

Comment:
Given that this is used only `instance.ts`, I don't see a point of this separate private file. 
Might as well move this to the `instance.ts` file until this is needed in more places.

Reply:
Done.